src/locales/index.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
+import languageList from './language-list'
+
+Vue.use(VueI18n)
+
+let messages = {}
+let languageOptions = []
+
+for (let i = 0; i < languageList.length; i++) {
+  messages[languageList[i].id] = languageList[i].message
+  languageOptions.push({
+    label: languageList[i].name,
+    value: languageList[i].id
+  })
+}
+
+const DEFAULT_LOCALE = 'en'
+
+// Create VueI18n instance with options
+const i18n = new VueI18n({
+  locale: DEFAULT_LOCALE, // set locale
```

Comment:
Setting fields on an object is never a good idea. Could we add a function like `changeLanguage` to change the internal state? This would make it easier to change the logic if we want to change things in the future.

Reply:
This is just a definition.

I set language during loading main or renderer process. See: 279ae61, 65e2be97.